obspy/core/util/headers.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+"""
+Base constants for ObsPy.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+# NO IMPORTS FROM OBSPY OR FUTURE IN THIS FILE! (file gets used at
+# installation time)
+import sys
+
+
+# XXX TODO when bumping to numpy 1.9.0: replace bytes() in io.reftek with
+# np.tobytes()
+HARD_DEPENDENCIES = {
+    "future": ">=0.12.4",
+    "numpy": ">=1.6.1",
+    "scipy": ">=0.9.0",
+    "matplotlib": ">=1.1.0",
+    "lxml": "",
+    "setuptools": "",
+    "sqlalchemy": "",
```

Comment:
If we do this separation of hard and soft dependencies we should also use the opportunity and make `sqlalchemy` optional - its only used for `obspy.db` but I think this is a rarely used module in practice for most users.